### PR TITLE
Fast form not broken.  Updates to fast displays.

### PR DESCRIPTION
**Pull request by CapnCrunchsna**

### CODE REVIEW

File: resources/views/index.blade.php [hunk 1]
```diff
@@ -14,6 +14,7 @@
     <link rel="stylesheet" type="text/css" href="angular/account/register.css">
     <link rel="stylesheet" type="text/css" href="angular/about/about.css">
     <link rel="stylesheet" type="text/css" href="angular/admin/privacy.css">
+    <link rel="stylesheet" type="text/css" href="angular/home/welcome.css">
```
Comment (marshall-davis) [inline resources/views/index.blade.php:17]: Yikes, we'll need some Gulp action to combine these or a reason they are all loaded but in separate files?

Comment (CapnCrunchsna) [reply resources/views/index.blade.php:17]: Yeah, there's a lot of them.  I could put all the css in one file, but I don't particularly like that either.

Comment (marshall-davis) [reply resources/views/index.blade.php:17]: This weekend I'll create a branch with better Gulp examples so you can see how it works if you've not used it. Will clean this right up.

Comment (CapnCrunchsna) [reply resources/views/index.blade.php:17]: Sure.  We use gulp at work, but I pretty much only use it, don't know how to set it up or really how it works.

**Pull request merged**